src/Components/README.md

```diff
@@ -29,42 +29,56 @@ The following contains a description of each sub-directory in the `Components` d
 
 ## Development Setup
 
-### Build
+**Note**: To build other specific projects from source, follow the instructions [on building the project](../../docs/BuildFromSource.md#step-3-build-the-repo).
 
-To build this specific project from source, follow the instructions [on building the project](../../docs/BuildFromSource.md#step-3-build-the-repo).
+### Building ASP.NET Core Components
 
-**Note:** You also need to run the preceding `build` command in the command line before building in VS to ensure that the Web.JS dependency is built.
+1. You'll need to install [Node](https://nodejs.org) on your machine.
 
-### A Guide to build Web.JS
+1. Ensure the repository is clean from any asset that could remain from previous version of the repository. This is recommended when switching branches, or after updating the working branch.
 
-Prior to building `src\Components\Web.JS\`
+```powershell
+git clean -xdff
+```
 
-1. You'll need to install [Node](https://nodejs.org) on your machine.
+You may need to kill some processes holding on files that are being deleted, like closing Visual Studio and other `msbuild` or `dotnet` processes. The following command may help you but be aware that this could also stop other important tasks:
```

Comment:
```suggestion
You may need to kill some processes holding on files that are being deleted, like closing Visual Studio and other `msbuild` or `dotnet` processes. There may also be lingering headless `chrome` processes, but they are not included in this command. The following command may help you but be aware that this could also stop other important tasks:
```